### DISCUSSION TITLE
Way to abstract crud actions

### DISCUSSION THREAD

**Discussion by Jarrodsz** [remix-run/remix#4380] [Proposals]
A standardized way to add edit and remove objects. Perhaps later with form scaffoling like rails and blitzjs(premature) has.

**Comment by sergiodxa** [comment]
This needs to be tied to a data source, Blitz and Rails can do it because they have a specific ORM, a Remix app can use any Node ORM, or fetch from an API, or even read from the fila system, there’s not a single data layer.
